### PR TITLE
update TiddlerInfo.multitids

### PR DESCRIPTION
**Pull request by BurningTreeC**
some typos and small changes for better understanding

### CODE REVIEW

File: languages/de-DE/TiddlerInfo.multids [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
+Advanced/ShadowInfo/OverriddenShadow/Hint: Der originale Schatten-Tiddler wurde durch diesen Tiddler überschrieben. Wenn Sie diesen Tiddler löschen, wird der originale Schatten-Tiddler wieder aktiv. Erstellen Sie vorher eventuell eine Sicherungskopie!
 Fields/Caption: Felder
 List/Caption: Liste
 List/Empty: Dieser Tiddler hat kein "list" Feld.
 Listed/Caption: Gelistet
 Listed/Empty: Dieser Tiddler wird nicht von anderen Tiddlern gelistet.
 References/Caption: Referenzen
-References/Empty: Kein Tiddler linkt zu Diesem.
+References/Empty: Kein Tiddler linkt zu diesem Tiddler.
 Tagging/Caption: Tagging
```
Comment (pmario) [inline languages/de-DE/TiddlerInfo.multids]: kommt das Wort "Tiddler" dann nicht zu oft vor?

Comment (BurningTreeC) [reply languages/de-DE/TiddlerInfo.multids]: I personally think that for starters it's easier to understand what's meant if you refer to the object more often. In my opinion it's better to sacrifice "good german" in favor of easier understanding

**Comment by pmario** [comment]
@Jermolene 
LGTM

File: languages/de-DE/TiddlerInfo.multids (hunk 1 above)
Comment (tobibeer) [reply languages/de-DE/TiddlerInfo.multids]: Ich bin für...

> Kein Tiddler linkt zu diesem.

No idea why *Diesem* was *Capitalized* before. It doesn't seem valid grammar. I imagine this was what bugged you more than anything.

**Comment by Jermolene** [comment]
Thanks @BurningTreeC

**Comment by BurningTreeC** [comment]
@Jermolene thanks for your great tool

File: languages/de-DE/TiddlerInfo.multids (hunk 1 above)
Comment (BurningTreeC) [reply languages/de-DE/TiddlerInfo.multids]: "Shatten Status" bugged me more. I'm fine with "Kein Tiddler linkt zu diesem" as I'm fine with "Kein Tiddler linkt zu diesem Tiddler" ... just a matter of taste

**Pull request merged**